react-native-in-app-payments-quickstart/app/screens/HomeScreen.tsx

```diff
@@ -608,14 +620,32 @@ export default function HomeScreen() {
     }
   };
 
+  const showValue = (title, description) => {
+    return (
+      <CommonAlert title={title} description={description} status={status} isVisible={showingDialogSheet} onDialogClick={onDialogClick} />
+    )
+  }
+
+  const setValue = (title, description, status) => {
```

Comment:
Do we have a better name fro this method?

Reply:
Done